### DISCUSSION TITLE
PR diff view doesn't seem to render tab characters

### DISCUSSION THREAD

**Discussion by francislavoie** [community/community#51875] [Mobile]
I'm on version `GitHub Mobile v1.107.0-beta (10119)`, Android 13.

Notice in the screenshot below, the non `+` lines have tabs (e.g. line 1330), but they don't align with the code with the `+` lines. This makes reading the diffs very difficult.

What's strange is sometimes the tabs do render, like the one before the `}` on line 1332.

![Screenshot_20230403-133906.png](https://user-images.githubusercontent.com/2111701/229585947-9fdd0bba-5b15-4b95-9891-7546fcf1557f.png)

Source is this PR: https://github.com/caddyserver/caddy/pull/5104

My code options configuration:

![Screenshot_20230403-134553.png](https://user-images.githubusercontent.com/2111701/229587162-a397b430-6f80-4e19-9be7-115772fe406e.png)

**Comment by iskae** [comment]
Hello @francislavoie 👋 

Thanks so much for reaching out ❤️ 

I have been able to reproduce this behaviour and I have created an internal bug issue for this 👍 We will try to fix it as soon as possible 🙇 

Please stay tuned for the future releases for the fix 😊 

Cheers!
